src/Package/Impl/PackageManager/PackageManagerWindowPane.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.ComponentModel.Design;
+using System.Runtime.InteropServices;
+using Microsoft.R.Components.History.Implementation;
+using Microsoft.R.Components.PackageManager;
+using Microsoft.R.Components.PackageManager.Implementation;
+using Microsoft.R.Components.PackageManager.ViewModel;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.R.Package.Commands;
+using Microsoft.VisualStudio.R.Package.Interop;
+using Microsoft.VisualStudio.R.Package.Shell;
+using Microsoft.VisualStudio.R.Packages.R;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.R.Package.PackageManager {
+    [Guid(WindowGuidString)]
+    internal class PackageManagerWindowPane : VisualComponentToolWindow<IRPackageManagerVisualComponent>, IOleCommandTarget {
+        private readonly IRPackageManager _packageManager;
+        public const string WindowGuidString = "363F84AD-3397-4FDE-97EA-1ABD73C64BB3";
+        public static Guid WindowGuid { get; } = new Guid(WindowGuidString);
+
+        private IOleCommandTarget _commandTarget;
+
+        public PackageManagerWindowPane(IRPackageManager packageManager) {
+            _packageManager = packageManager;
+            Caption = Resources.PackageManagerWindowCaption;
+        }
+
+        protected override void OnCreate() {
+            _commandTarget = new CommandTargetToOleShim(null, Component.Controller);
```

Comment:
Problem with local controller is that commands are not visible to the top level menu and can only appear as tool window toolbar buttons. For Plot commands I had to redo local commands to PackageCommand (see plot window pane) so they can be visible in the menu